cmd/process-agent/collector.go

```diff
@@ -125,6 +125,12 @@ func (l *Collector) runCheck(c checks.Check) {
 		log.Errorf("Unable to run check '%s': %s", c.Name(), err)
 		return
 	}
+
+	if result == nil {
```

Comment:
Just for my understanding, in which cases a check would return an empty result?

Reply:
There are a handful of spots - 
https://github.com/DataDog/datadog-agent/blob/main/pkg/process/checks/container.go#L98
https://github.com/DataDog/datadog-agent/blob/main/pkg/process/checks/container_rt.go#L75

Reply:
Thanks for clarifying it!